src/components/view-pane/index.tsx

```diff
@@ -4,26 +4,27 @@ import * as React from 'react';
 import * as CSSModules from 'react-css-modules';
 import {connect} from 'react-redux';
 import {Data} from 'vega-lite/build/src/data';
-
+import {Transform} from 'vega-lite/build/src/transform';
 import {ActionHandler, createDispatchHandler} from '../../actions/redux-action';
 import {ShelfAction} from '../../actions/shelf';
 import {State} from '../../models';
 import {extractPlotObjects, PlotObject} from '../../models/plot';
 import {hasWildcards} from '../../models/shelf/spec';
-import {getData, getMainResult, getQuery} from '../../selectors';
+import {getData, getFilters, getMainResult, getQuery} from '../../selectors';
 import {Plot} from '../plot';
 import {PlotList} from '../plot-list';
 import * as styles from './view-pane.scss';
 
 export interface ViewPaneProps extends ActionHandler<ShelfAction> {
   data: Data;
   query: Query;
+  filters: Transform[];
```

Comment:
Instead of all possible transform, this could be just a subset of transform that we support.